### DISCUSSION TITLE
Transfert entre pod

### DISCUSSION THREAD

**Discussion by RomainCahuNormandie** [EsupPortail/Esup-Pod#762] [Ideas]
Bonjour, 

On nous demande de tranférer manuellement toutes nos vidéos de nos anciens pod académiques au pod national. N'y aurait-il pas un moyen de le faire automatiquement ? La tâche est sacrément fastidieuse quand on a des dizaines de vidéos de posées...

**Comment by ptitloup** [comment]
Bonjour,
Cette question est également traitée par la DNE. Je viens de leur transmettre votre demande.
Bien cordialement

**Comment by BHoury** [comment]
Bonjour,

Il n'y a pas actuellement de process automatique pour cette migration. Cela est assez complexe car le Pod Normandie et PodEduc ne sont pas dans les mêmes versions et l'organisation (chaînes, thèmes, ...) est différente.

Cordialement

**Reply by RomainCahuNormandie** [reply]
Je comprends que cela puisse être compliqué, mais peut-être pas impossible ? Pour en débattre avec quelques autres enseignants de l'académie impliqués dans le numérique, la seule chose qu'il risque d'arriver c'est la disparition pure et simple d'une énorme partie des contenus et la migration sur Youtube...
